commands/auto/twoblhangerbounce.py

```diff
@@ -12,7 +13,7 @@
 from commands.followtrajectory import FollowTrajectory
 from commands.indexer.feedforward import FeedForward
 from commands.indexer.holdball import HoldBall
-from commands.resetgyro import ResetGyro
+from commands.resetdrive import ResetDrive
```

Comment:
Good change but this should get deleted anyway. More importantly, we have to figure out why auto isn't working (again my guess is odometry stuff, maybe we can disable the vision assisted stuff during auto?)

Reply:
I wonder how we would be able to test this well between now and next competition...

Reply:
If we have to we can go outside. At the very least we need it working as expected in sim and then we could try it on the practice field

Reply:
well currently it does work right in simulation correct? We should test to see if our odometry calculations are messing up autonomous by going outside.

Reply:
Not quite, it still does the weird stuff in sim. For now we can merge it anyway and delete that file later i guess